### DISCUSSION TITLE
监控 metrics, grafana

### DISCUSSION THREAD

**Discussion by wkai886** [alibaba/ilogtail#499] [Help] [answered]
ilogtail 有暴露 metrics嘛，如何配置grafana

**Comment by yyuuttaaoo** [comment, marked as answer]
是指ilogtail自身的运行状况吗？
目前本地日志中有暴露一些metric，可以看本地的snapshot目录。
上报功能目前还没有，有一个相关的issue，欢迎讨论共建 https://github.com/alibaba/ilogtail/issues/75

**Comment by messixukejia** [comment]
关注下下次的双周会，到时可以一起交流下需求。
